site/_layouts/doc.liquid

```diff
@@ -286,15 +286,20 @@
                   </ul>
                 </li>
                 <li {% if page.url contains "/account-management/" %} class="open"{% endif %}>
-                  <a href="#" class="sub-menu"><i class="fal fa-fw fa-bullseye"></i> Self Service Account Mgmt<i class="fal fa-chevron-{% if page.url contains "/account-management/" %}up{% else %}down{% endif %} fa-fw"></i></a>
+                  <a href="#" class="sub-menu"><i class="fal fa-fw fa-users"></i> End User UI<i class="fal fa-chevron-{% if page.url contains "/account-management/" %}up{% else %}down{% endif %} fa-fw"></i></a>
                   <ul>
-                    <li {% if page.url == "/docs/v1/tech/account-management/" %}class="active"{% endif %}><a href="/docs/v1/tech/account-management/" >Overview</a></li>
+                    <li {% if page.url contains "/account-management/" %} class="open"{% endif %}>
+                      <a href="#" class="sub-menu">Multi-Factor Auth (MFA)<i class="fal fa-chevron-{% if page.url contains "/account-management/" %}up{% else %}down{% endif %} fa-fw"></i></a>
+                      <ul>
+                        <li {% if page.url == "/docs/v1/tech/account-management/" %}class="active"{% endif %}><a href="/docs/v1/tech/account-management/" >Overview</a></li>
+                        <li {% if page.url == "/docs/v1/tech/account-management/authenticator-two-factor/" %}class="active"{% endif %}><a href="/docs/v1/tech/account-management/authenticator-two-factor/">Add Authenticator Factor</a></li>
+                        <li {% if page.url == "/docs/v1/tech/account-management/email-two-factor/" %}class="active"{% endif %}><a href="/docs/v1/tech/account-management/email-two-factor/">Add Email Factor</a></li>
+                        <li {% if page.url == "/docs/v1/tech/account-management/sms-two-factor/" %}class="active"{% endif %}><a href="/docs/v1/tech/account-management/sms-two-factor/">Add SMS Factor </a></li>
+                        <li {% if page.url == "/docs/v1/tech/account-management/customizing-account-management/" %}class="active"{% endif %}><a href="/docs/v1/tech/account-management/customizing-account-management/">Customizing</a></li>
+                        <li {% if page.url == "/docs/v1/tech/account-management/troubleshooting/" %}class="active"{% endif %}><a href="/docs/v1/tech/account-management/troubleshooting/">Troubleshooting</a></li>
+                      </ul>
+                    </li>
                     <li {% if page.url == "/docs/v1/tech/account-management/updating-user-data/" %}class="active"{% endif %}><a href="/docs/v1/tech/account-management/updating-user-data/">Updating User Data & Password</a></li>
```

Comment:
I think it makes sense to have the title in the nav and the title in the header consistent for this page, but they are not. can you make them consistent, pls.